templates/publisher/listing.html

```diff
@@ -5,218 +5,220 @@
-                <input
-                  type="radio"
-                  name="private"
-                  id="is-private"
-                  {% if private %}
-                    checked="checked"
-                  {% endif %}
-                  value="private" />
-                <label class="u-no-margin--top" for="is-private">Private
-                  <span class="p-form-help-text">Snap is hidden in stores and only accessible by the publisher and collaborators</span>
-                </label>
-              </div>
+            <div class="row">
+              <input
+                type="radio"
+                name="private"
+                id="is-private"
+                {% if private %}
+                  checked="checked"
+                {% endif %}
+                value="private" />
+              <label for="is-private">Private
+                <span class="p-form-help-text">Snap is hidden in stores and only accessible by the publisher and collaborators</span>
+              </label>
             </div>
           </div>
+        </div>
 
+        <div class="p-strip is-shallow">
           <div class="row">
-            <div class="col-12">
-              <hr />
-            </div>
+            <hr class="u-no-margin--bottom" />
           </div>
+        </div>
 
-          <div class="row">
-            <div class="col-2">
-              <label class="p-label--grid-baseline">Summary: </label>
-            </div>
-            <div class="col-8">
-              <div class="p-form-validation u-no-margin {% if field_errors and field_errors['summary'] %}is-error{% endif %}">
-                <div class="p-form-validation__field">
-                  <input class="p-form-validation__input" type="text" name="summary" value="{{ summary }}" required maxlength="128"/>
-                </div>
-                {% if field_errors and field_errors['summary'] %}
-                  <p class="p-form-validation__message">
-                    <strong>Error:</strong> {{ field_errors['summary'] }}
-                  </p>
-                {% endif %}
+        <div class="row">
+          <div class="col-2">
+            <label>Summary: </label>
+          </div>
+          <div class="col-8">
+            <div class="p-form-validation u-no-margin {% if field_errors and field_errors['summary'] %}is-error{% endif %}">
+              <div class="p-form-validation__field">
+                <input class="p-form-validation__input" type="text" name="summary" value="{{ summary }}" required maxlength="128"/>
               </div>
+              {% if field_errors and field_errors['summary'] %}
+                <p class="p-form-validation__message">
+                  <strong>Error:</strong> {{ field_errors['summary'] }}
+                </p>
+              {% endif %}
             </div>
           </div>
-          <div class="row">
-            <div class="col-2">
-              <label class="p-label--grid-baseline">Description: </label>
```

Comment:
If `p-label--grid-baseline` is not needed anymore it should be removed from CSS as well.

Reply:
Ah, thought it was part of vanilla, didn't realise we had an override. Removed!